sk0-front/src/app/common/services/registerEmailValidator.ts

```diff
@@ -0,0 +1,26 @@
+import { AbstractControl, ValidationErrors, ValidatorFn } from "@angular/forms";
+import { Observable, of } from "rxjs";
```

Comment:
This import is not used (as you can see it in the sonarscanner output), you can delete it.